libeos-payg/errors.h

```diff
@@ -32,6 +32,8 @@ G_BEGIN_DECLS
  * @EPG_MANAGER_ERROR_TOO_MANY_ATTEMPTS: Too many attempts to verify a code
  *    in recent history.
  * @EPG_MANAGER_ERROR_DISABLED: Pay as you go is disabled.
+ * @EPG_MANAGER_ERROR_DISPLAY_ACCOUNT_ID: The given code was valid, but instead of
+ * adding time, tell the listener to show the account ID assigned to the machine.
```

Comment:
indentation is off here

Reply:
Also, I think you should update the docs for `epg_provider_add_code()` to mention that this error can be returned and should be handled.